Header header
int32 id
string name
string command
string group
string[] dependencies